frontend/dataTab.jsx

```diff
@@ -28,27 +28,31 @@ function renderData({ item }, navigation) {
         <DataPointView>
           <BodyText>
             Duration:
+            {' '}
             {item.duration}
```

Comment:
A cleaner way would be to use string interpolation, but it doesn't really make a difference tbh
```suggestion
            {` ${item.duration}`}
```

Reply:
@vinaynagappala lol I only suggested for one of the elements but if you're gonna apply it you should apply it to all of them, both in data tab and detailed data page, for consistency